tests/kernel/install_ltp.pm

```diff
@@ -238,69 +238,6 @@ sub install_from_git {
               . get_ltp_version_file()));
 }
 
-sub add_ltp_repo {
-    my $repo = get_var('LTP_REPOSITORY');
-
-    if (!$repo) {
-        if (is_sle || is_transactional) {
-            add_qa_head_repo;
-            return;
-        }
-
-        # ltp for leap15.2 is available only x86_64
-        if (is_leap('15.4+')) {
-            $repo = get_var('VERSION');
-        } elsif ((is_leap('=15.2') && is_x86_64) || is_leap('15.3+')) {
-            $repo = sprintf("openSUSE_Leap_%s", get_var('VERSION'));
-        } elsif (is_tumbleweed) {
-            $repo = "openSUSE_Factory";
-            $repo = "openSUSE_Factory_ARM" if (is_aarch64() || is_arm());
-            $repo = "openSUSE_Factory_PowerPC" if is_ppc64le();
-            $repo = "openSUSE_Factory_zSystems" if is_s390x();
-        } else {
-            die sprintf("Unexpected combination of version (%s) and architecture (%s) used", get_var('VERSION'), get_var('ARCH'));
-        }
-        $repo = "https://download.opensuse.org/repositories/benchmark:/ltp:/devel/$repo/";
-    }
-
-    zypper_ar($repo, name => 'ltp_repo');
-}
-
-sub get_default_pkg {
-    my @packages;
-
-    if (is_sle && is_released) {
-        push @packages, 'ltp-stable';
-        push @packages, 'ltp-stable-32bit' if is_x86_64;
-    } else {
-        push @packages, 'ltp';
-        push @packages, 'ltp-32bit' if is_x86_64 && !is_jeos;
-    }
-
-    return join(' ', @packages);
-}
-
-sub install_from_repo {
-    my @pkgs = split(/\s* \s*/, get_var('LTP_PKG', get_default_pkg));
```

Comment:
I now see that you're just moving this code. Maybe it makes still sense to improve this section as I mentioned.